lms/templates/components/header/header.underscore

```diff
@@ -0,0 +1,14 @@
+<header class="page-header has-secondary">
+  <div class="header-main">
+    <% if (breadcrumbs !== null && breadcrumbs.length > 0) { %>
+    <nav class="breadcrumbs">
```

Comment:
We generally indent under an ``if`` to make the extent of the conditional clear. It means that the indentation isn't what we want in the HTML, but that doesn't really matter by the time we dynamically insert it into the DOM.